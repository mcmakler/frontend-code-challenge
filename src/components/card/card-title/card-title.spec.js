import React from 'react';
import CardTitle from './card-title';
import ReactTestRenderer from 'react-test-renderer';

describe('CardTitle specs', () => {
  it('should match snapshot', () => {
    const component = ReactTestRenderer.create(<CardTitle title='card title'/>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
