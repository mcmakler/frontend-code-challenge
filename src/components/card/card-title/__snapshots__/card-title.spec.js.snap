// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle specs should match snapshot 1`] = `
<div
  className="cardTitle"
>
  card title
</div>
`;
